src/components/Iconography/Iconography.stories.mdx

```diff
@@ -0,0 +1,176 @@
+
+<Preview>
+  <Story name="Icon Examples">
+    <span style={{ marginRight: "1rem" }}>
+      <Icon
+        decorative
+        size={IconSizes.xxxlarge}
+        name="check"
+        style={{ marginRight: "1rem" }}
+      />
+    </span>
+    <span style={{ marginRight: "1rem" }}>
+      <Icon
+        decorative
+        size={IconSizes.xxxlarge}
+        name="close"
+        style={{ marginRight: "1rem" }}
+      />
+    </span>
+    <span style={{ marginRight: "1rem" }}>
+      <Icon
+        decorative
+        size={IconSizes.xxxlarge}
+        name="error_filled"
+        style={{ marginRight: "1rem" }}
+      />
+    </span>
+    <span style={{ marginRight: "1rem" }}>
+      <Icon
+        decorative
+        size={IconSizes.xxxlarge}
+        name="speaker_notes"
+        style={{ marginRight: "1rem" }}
+      />
+    </span>
+    <span style={{ marginRight: "1rem" }}>
+      <Icon
+        decorative
+        size={IconSizes.xxxlarge}
+        name="logo_nypl"
+        style={{ marginRight: "1rem" }}
+      />
+    </span>
+    <span style={{ marginRight: "1rem" }}>
+      <Icon decorative size={IconSizes.xxxlarge} name="logo_brooklyn" />
+    </span>
+  </Story>
+</Preview>
+
+The DS utilizes the `Filled` icons available in the [Material UI Design System](https://material-ui.com/components/material-icons/).
+
+### Size
+
+Icons in NYPL projects can be rendered at the following sizes.
+
+<Preview>
+  <Story name="Size">
+    <div style={{ display: "flex", alignItems: "center" }}>
+      <span style={{ marginRight: "1rem" }}>
+        <Icon decorative size={IconSizes.small} name="check" />
+      </span>
+      <span style={{ marginRight: "1rem" }}>
+        <Icon decorative size={IconSizes.medium} name="check" />
+      </span>
+      <span style={{ marginRight: "1rem" }}>
+        <Icon decorative size={IconSizes.large} name="check" />
+      </span>
+      <span style={{ marginRight: "1rem" }}>
+        <Icon decorative size={IconSizes.xlarge} name="check" />
+      </span>
+      <span style={{ marginRight: "1rem" }}>
+        <Icon decorative size={IconSizes.xxlarge} name="check" />
+      </span>
+      <span style={{ marginRight: "1rem" }}>
+        <Icon decorative size={IconSizes.xxxlarge} name="check" />
+      </span>
+    </div>
+  </Story>
+</Preview>
+
+| Icon Size  | CSS Value         | CSS Variable  |
+| ---------- | ----------------- | ------------- |
+| `small`    | `0.875rem` (14px) | n/a           |
+| `medium`   | `1.125rem` (18px) | n/a           |
+| `large`    | `1.5rem` (24px)   | `--space-m`   |
+| `xlarge`   | `2rem` (32px)     | `--space-l`   |
+| `xxlarge`  | `3rem` (48px)     | `--space-xl`  |
+| `xxxlarge` | `4rem` (64px)     | `--space-xxl` |
+
+### Color
+
+Icons in NYPL projects should be rendered with one color. Color options are built into the Icon component.
```

Comment:
Can you include that the colors are available through the `IconColors` enum?

Reply:
I added additional verbiage about the enum values.